ContactDetailsApi/V2/Factories/EntityFactory.cs

```diff
@@ -14,10 +14,13 @@ public static class EntityFactory
     {
         public static ContactDetails ToDomain(this ContactDetailsEntity databaseEntity)
         {
+            if (databaseEntity == null) return null;
+            if (databaseEntity.ContactInformation == null) return null;
```

Comment:
If these are intended to both return null, it could be combined into a single expresion:

```cs
if (databaseEntity == null || databaseEntity.ContactInformation == null) return null;
```